src/scripts/productsTabs.js

```diff
@@ -0,0 +1,60 @@
+const furnitureCategoryTabs = document.querySelectorAll('.category-tab-select');
+
+furnitureCategoryTabs.forEach(function (tab) {
+  tab.addEventListener('click', checkResolutionOfScreen);
+});
+
+function checkResolutionOfScreen (event) {
+  const targetTab = event.currentTarget.getAttribute('href');
+  const tabPanelId = targetTab.substring(1, targetTab.length);
+  const tabPanel = document.getElementById(tabPanelId);
+  const tabPanelProductElements = tabPanel.children[0].children.length;
+
+  const width = window.innerWidth;
+  const height = window.innerHeight;
+  let orientation;
+
+  if (width > height) {
+    orientation = 'landscape';
+  } else if (width < height) {
+    orientation = 'portrait';
+  }
+
+  detectScreenMode(orientation, width, tabPanelProductElements);
+}
+
+function createDots (number) {
+  var dotMenu = document.querySelector('#dots-container');
+
+  while (dotMenu.firstChild) {
+    dotMenu.innerHTML = '';
+  }
+
+  for (let i = 0; i < number; i++) {
+    var liElement = document.createElement('li');
+    var aElement = document.createElement('a');
+    aElement.setAttribute('href', '#');
+    liElement.appendChild(aElement);
+    dotMenu.appendChild(liElement);
+  }
+}
+
+function detectScreenMode (orientation, width, tabPanelProductElements) {
+  const landscapeLargeMode = orientation === 'landscape' && width > 1024;
+  const landscapeMiddleMode = orientation === 'landscape' && width > 995 < 1024;
+  const landscapeSmallMode = orientation === 'landscape' && width > 568 < 768;
```

Comment:
moim zdaniem zapis `width > 568 < 768` nie działa. podziel to na dwa bloki `width > ...  && width < ...`